### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by yoavbls** [yoavbls/pretty-ts-errors#120] [Announcements]
## What's Changed
* Extend list of known error numbers by @bennycode in https://github.com/yoavbls/pretty-ts-errors/pull/115
* refactor: use PropertyKey instead of key list by @EdamAme-x in https://github.com/yoavbls/pretty-ts-errors/pull/113
* Fix: adjust missing props regex to work for private properties by @kevinramharak in https://github.com/yoavbls/pretty-ts-errors/pull/106
* Fix issue #82 - ts(4113), ts(4117) by @yoavbls in https://github.com/yoavbls/pretty-ts-errors/pull/114
* Add `ts-plugin` to diagnostic sources by @yoavbls in https://github.com/yoavbls/pretty-ts-errors/pull/103
* update @vscode/test-electron to 2.3.9 by @kevinramharak in https://github.com/yoavbls/pretty-ts-errors/pull/99
* fix npm: watch problem matcher stalling default debug task by @kevinramharak in https://github.com/yoavbls/pretty-ts-errors/pull/112
* Document the custom CSS hack by @yoavbls in https://github.com/yoavbls/pretty-ts-errors/pull/119
* Upgrade dependencies by @yoavbls in https://github.com/yoavbls/pretty-ts-errors/pull/116
* Bump braces from 3.0.2 to 3.0.3 by @dependabot in https://github.com/yoavbls/pretty-ts-errors/pull/111

**Full Changelog**: https://github.com/yoavbls/pretty-ts-errors/compare/v0.5.3...v0.6.0

# In Memoriam ❤️

This release is dedicated to Ice, my beloved dog.
May your memory continue to inspire and bring joy.

Rest in peace, Ice. 🐾
<img src="https://github.com/user-attachments/assets/00629543-b7b2-45ae-bd20-292eef8d113a" width="600" />

<hr /><em>This discussion was created from the release <a href='https://github.com/yoavbls/pretty-ts-errors/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by jmezzacappa** [comment]
Beautiful dog. I'm sorry for your loss.

**Reply by yoavbls** [reply]
Thank you ❤️
